pkg/audit/verifier.go

```diff
@@ -396,6 +409,17 @@ func (verifier *Verifier) Reverify(ctx context.Context, stripe *Stripe) (report
 				ch <- result{nodeID: piece.NodeId, status: failed}
 				verifier.log.Debug("Reverify: hashes mismatch (audit failed)", zap.Stringer("Node ID", piece.NodeId),
 					zap.Binary("expected hash", pending.ExpectedShareHash), zap.Binary("downloaded hash", downloadedHash))
+				//remove failed audit pieces from the pointer so as to only penalize once for failed audits
```

Comment:
to keep this consistent with the other "fail" case in `Reverify`, move your changes above the `ch <- result{nodeID: piece.NodeId, status: failed}` on line 409. This will also ensure that your success case (currently on line 415) properly occurs if applicable. Otherwise, you have already marked this node as failed